hibernate-core/src/main/java/org/hibernate/engine/jdbc/connections/internal/ConnectionProviderInitiator.java

```diff
@@ -50,6 +56,7 @@
  * @author Steve Ebersole
  * @author Brett Meyer
  */
+@SuppressWarnings( {"deprecation", "unchecked"} )
```

Comment:
Please don't suppress warnings on a whole class like this.

Reply:
done.